### PR TITLE
frontend changes

### PR DESCRIPTION
**Pull request by chirag-ghosh**
# Description

## Type of change

- [x] Move from npm to pnpm
- [x] Restrict website to users connected to IIT Kharagpur network

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,38 +1,62 @@
...
+  }, []);
 
-        <EmptyRoomsTable schedule={this.props.empty_schedule} />
-        {Object.keys(this.props.schedule).map(key => {
-          return <CustomTable room={key} schedule={this.props.schedule[key]} />;
-        })}
+  if (loading)
+    return (
+      <div className="message-banner">
+        <iframe src="https://lottie.host/?file=ca9a3787-e6db-4878-a0d3-d10dde95b225/MjnHw4B5gw.json"></iframe>
```
Comment (harshkhandeparkar) [inline frontend/src/App.js:26]: Is it safe to load an iframe (is it vulnerable to man in the middle)? Isn't it possible to embed this image directly?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,38 +1,62 @@
...
+        <div className="message">Chillzone is loading</div>
       </div>
     );
-  }
+  else if (!show)
+    return (
+      <div className="message-banner">
+        <iframe src="https://lottie.host/?file=46b58d32-043e-4268-94cd-4d7817565000/xNnhIYFoYA.json"></iframe>
```
Comment (harshkhandeparkar) [inline frontend/src/App.js:33]: Same here

File: frontend/src/App.js (hunk 1 above)
Comment (chirag-ghosh) [reply frontend/src/App.js:26]: [Not really](https://stackoverflow.com/questions/7289139/why-are-iframes-considered-dangerous-and-a-security-risk). 
Yes (problem is, it's not an image. will try to download gif and use them).

Comment (harshkhandeparkar) [reply frontend/src/App.js:26]: > it's not an image
:hmm:

**Review by harshkhandeparkar — APPROVED** [review]

**Pull request merged**